src/current/_includes/v23.1/sidebar-data/migrate.json

```diff
@@ -61,29 +84,6 @@
         }
       ]
     },
-    {
-      "title": "Migrate with Third-Party Tooling",
-      "items": [
-        {
-          "title": "AWS DMS",
-          "urls": [
-            "/${VERSION}/aws-dms.html"
-          ]
-        },
-        {
-          "title": "Qlik Replicate",
-          "urls": [
-            "/${VERSION}/qlik.html"
-          ]
-        },
-        {
-          "title": "Striim",
-          "urls": [
-            "/${VERSION}/striim.html"
-          ]
-        }
-      ]
-    },
     {
       "title": "Migrate from Serverless to Dedicated",
```

Comment:
We really need to simplify the title on the page this targets -- "Migrate from a CockroachDB Serverless to CockroachDB Dedicated Cluster". Can it just be "Migrate from Serverless to Dedicated"?